pkg/operation/shoot/shoot.go

```diff
@@ -355,15 +355,13 @@ func (s *Shoot) UpdateInfoStatus(ctx context.Context, c client.Client, useStrate
 	return nil
 }
 
-// GetExtensionComponentsForMigration returns a list of component.DeployMigrateWaiters of extension components that
-// should be migrated by the shoot controller.
-func (s *Shoot) GetExtensionComponentsForMigration() []component.DeployMigrateWaiter {
+// GetExtensionComponentsForParallelMigration returns a list of component.DeployMigrateWaiters of extension components that
+// should be migrated by the shoot controller in parallel.
```

Comment:
You should probably state that `ControlPlane` and `Infrastructure` are not returned here and must be handled elsewhere.